web/pdf_page_view.js

```diff
@@ -513,6 +513,22 @@ class PDFPageView {
     }
   }
 
+  /**
+   * @typedef {Object} PDFPageViewUpdateParameters
+   *
+   * @property {number} [scale] The new scale, if specified.
+   * @property {number} [rotation] The new rotation, if specified.
+   * @property {Promise<OptionalContentConfig>} [optionalContentConfigPromise]
+   *   A promise that is resolved with an {@link OptionalContentConfig} instance.
+   *   The default value is `null`.
+   * @property {number} [drawingDelay]
+   */
+
+  /**
+   * Update page view by eitheir modifiying scale or rotation.
+   *
+   * @param {PDFPageViewUpdateParameters}
```

Comment:
Note that this method does more than only changing the scale/rotation, hence my suggestion:
```suggestion
   * Update e.g. the scale and/or rotation of the page.
   * @param {PDFPageViewUpdateParameters}
```